### ISSUE TITLE
Accessibility issue: Text elements should have a reasonable contrast ratio (6)

### ISSUE THREAD

**Issue by ameliejyc** [mineshmshah/mineshmshah.github.io#1]
See this link for more details:

https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_color_01

Applies to the following code:

<span class=​"hint">​
"Welcome to my github page!"
<br>​
" My name is Minesh and here'sa few words!"
</span>​
<span class=​"hint">​
"Just a quick summary:"
<br>​
" Ex-banker, Musician, Coder, Designer, Traveller"
</span>​
<span class=​"title">​Ideas​</span>​
<span class=​"hint">​
"I have a head full of ideas"
<br>​
" And want to bring it alive with code"
</span>​
<span>​Thanks for checking this out! ​</span>​
<a href=​"https:​/​/​mineshmshah.github.io" target=​"_blank">​Pease check out more here!"​</a>​
